frontend/src/pages/Mesh/MeshElems.tsx

```diff
@@ -253,7 +254,7 @@ export const elems = (c: Controller): { edges: Edge[]; nodes: Node[] } => {
 
 // TODO: When/if it is fixed this can be replaced with a straight call to node.getAllNodeChildren();
 // https://github.com/patternfly/patternfly-react/issues/8350
-export const descendents = (node: Node): Node[] => {
+export const descendents = <E extends NodeModel = NodeModel, D = any>(node: Node<E, D>): Node<E, D>[] => {
```

Comment:
This method is duplicated for the Patternfly graph. We should apply the same type, or, even better, use the same Patternfly helpers for both graphs (traffic and mesh) :slightly_smiling_face:

Reply:
I moved this method to `helpers/GraphHelpers.tsx`. Not sure if that's the best place for it but at least it's now shared between the two graph pages. There's other methods that look like they are shared between the two graphs that could move there as well but I left those untouched in this PR.

Reply:
The history here is just that the PFT traffic graph was still in development when we started the mesh page, so I just copied the whole thing as a starting point for Mesh.  I don't know if we want to go down that path of unifying PFT utilities, but I'm sort of -1 on just partially moving one util. In general I'm not too aggressive about avoiding code duplication because I like compartmentalized feature code.  It's easier to tweak and locate as needed, and I'm typically willing to deal with the fix-in-multiple-places as a trade-off.  Having said that, there are definitely helpers in MeshElems and ElemsPF that are purely at the PFT level, and could be candidates for sharing.

Reply:
@jshaughn so what would you prefer here? Move this method back into the individual graphs and copy the type signature update to both _or_ copy the rest of the helper methods to `graph/helpers`? I'm fine with either.

Reply:
OK, since you started, let's just start consolidating.  Take the entire "// PFT Helpers section" from GraphPFElems.tsx and put it in the new GraphHelpers. This version of the code has a fix needed for traffic graph that should be irrelevant to Mesh graph.  Delete that whole section from MeshElems.tsx.

Reply:
Yes, this is a trade-off between isolation and avoiding code duplication. I understand that code duplicated across both meshes could change in the future, but I’m +1 on unifying helpers and utilities that are exclusively related to Patternfly topology and will always apply to both graphs. We could create a new issue/PR, moving all helpers is out of scope of this PR.

Reply:
Alright I moved all the helpers.